presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConnectorFactory.java

```diff
@@ -13,21 +13,26 @@
  */
 package io.prestosql.plugin.hive;
 
+import com.google.common.base.Throwables;
 import io.prestosql.spi.connector.Connector;
 import io.prestosql.spi.connector.ConnectorContext;
 import io.prestosql.spi.connector.ConnectorFactory;
 import io.prestosql.spi.connector.ConnectorHandleResolver;
 
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 import java.util.Map;
 import java.util.Optional;
 
 import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.base.Strings.isNullOrEmpty;
-import static io.prestosql.plugin.hive.InternalHiveConnectorFactory.createConnector;
 
 public class HiveConnectorFactory
         implements ConnectorFactory
 {
+    private static final String HIVE_CONNECTOR_FACTORY = InternalHiveConnectorFactory.class.getName();
```

Comment:
I'd make this a local variable so it's next to the other code. Seems easier to read that way.